native-image/embed-sbom/run.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -ex
+
+native-image -Ob --enable-sbom=cyclonedx -m jdk.httpserver -o jwebserver
+
+# Requires GraalVM for JDK 24 Early Access build 24.ea.23-graal or later:
+# sdk install java 24.ea.23-graal
+cd sbom-test
+mvn clean package
+mvn -Pnative package
```

Comment:
This step will pass with the new EA build available. Currently it's on `24.ea.22-graal`. We need `24.ea.23-graal` for the new option `--enable-sbom=class-level` to become available.

Reply:
Noted.